doc/source/release-notes/Percona-XtraDB-Cluster-8.0.22-13.1.rst

```diff
@@ -0,0 +1,35 @@
+.. _PXC-8.0.22-13.1:
+
+================================================================================
+*Percona XtraDB Cluster* 8.0.22-13.1
+================================================================================
+
+:Date: February 24, 2021
+:Installation: `Installing Percona XtraDB Cluster <https://www.percona.com/doc/percona-xtradb-cluster/8.0/install/index.html>`_
+
+Improvements
+================================================================================
+
+* :jirabug:`PXC-3115`: SELinux and AppArmor policy
+
+
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PXC-3501`: Modify wsrep_row_upd_check_foreign_constraints() to include foreign key dependencies in the writesets for DELETE query (Thanks to user Steven Gales for reporting this issue)
+* :jirabug:`PXC-3502`: Corrected condition in thd_binlog_format() function for List Index process (Thanks to user Paweł Bromboszcz for reporting this issue)
+* :jirabug:`PXC-2913`: Correct MDL locks assertion when wsrep provider is unloaded
+* :jirabug:`PXC-3475`: Adjust mysqld_safe script to parse the 8.0 log style properly
+
+
+
+Known Issues (unfixed problems that you should be aware of)
+================================================================================
+
+* :jirabug:`PXC-3039`: No useful error messages if an SSL-disabled node tries to join an SSL-enabled cluster
+* :jirabug:`PXC-3092`: Log a warning at startup if a keyring is specified, but cluster traffic encryption is turned off
+* :jirabug:`PXC-3093`: Completed SST Transfer incorrectly logged by garbd (Timing is not correct)
+* :jirabug:`PXC-3159`: Modify the error handling to close the communication channels and abort the joiner node when the donor crashes
```

Comment:
PXC-3159 was fixed in 8.0.20-11 and is still marked as known issue here. I think we should remove it from this list.

Reply:
Removed when updated.